dpnp/dpnp_iface_statistics.py

```diff
@@ -244,11 +245,10 @@ def cov(x1, y=None, rowvar=True, bias=False, ddof=None, fweights=None, aweights=
 
     Limitations
     -----------
-    Input array ``m`` is supported as :obj:`dpnp.ndarray`.
-    Dimension of input array ``m`` is limited by ``m.ndim > 2``.
+    Input array ``x1`` is supported as :obj:`dpnp.ndarray`.
```

Comment:
I think the docstring should continue using ``m`` (for matrix), but perhaps to fix the disconnect the docstring should start with the usage line, e.g. https://github.com/IntelPython/dpctl/blob/master/dpctl/tensor/_search_functions.py#L43